Islandora-CLAW/Alpaca
# Merge broadcaster and pipeline

**GitHub Issue**: Resolves https://github.com/Islandora-CLAW/CLAW/issues/569
# What does this Pull Request do?

Merges the broadcaster and pipeline code to be a single route, where behavior is controlled via a header that gets converted to an `ExchangePattern`.

# What's new?
We're using the routingSlip pattern for both now.  Using an exchange pattern of `InOnly` causes the same message to get passed to each recipient.  Processing occurs in parallel.  Using the exchange pattern of `InOut` causes the output of each recipient to get passed to the next, with processing occuring in sequence.  In other words, `InOut` causes each step to 'block', allowing us to control the sequence of steps in a workflow.

If no header is provided for the exchange pattern, the default behaviour is to use `InOnly`.

# How should this be tested?

- Pull these changes into Alpaca
- `./gradlew install`
- `/opt/karaf/bin/client`
- `bundle:list | grep islandora` to get the number of the `islandora-connector-broadcast` bundle 
- `bundle:update THAT_NUMBER`

Everything should continue to work as expected.  That is, go do some CRUD on an entity and the changes should be reflected in the triplestore.

# Additional Notes:
Thanks to @Natkeeran for posing the right questions when working through an earlier issue, as that's what prompted this improvement.

# Interested parties
@Islandora-CLAW/committers
